### ISSUE TITLE
Unable to find any GraphQL type definitions for the following pointers

### ISSUE THREAD

**Issue by batmanhit** [kamilkisiela/graphql-inspector#2001]
**Describe the bug**

`Unable to find any GraphQL type definitions for the following pointers`

The following errors out when use `wildcard`.
```ssh
graphql-inspector validate ./client/**/*.graphql  ./schema.graphql
```

Although when I copy/paste all the `graphql` queries/mutations into `document.graphql`, it never errors out.
```ssh
graphql-inspector validate ./client/documents.graphql  ./schema.graphql
```

**To Reproduce**

1. First Approach

`client/queries/companies.graphql`
```
query getCompanies {
  companies {
    id
  }
}
```

`client/queries/hotels.graphql`
```
query getHotels {
  hotels {
    id
  }
}
```

`schema.graphql`
```
type Company {
  id: ID!
}

type Hotel {
  id: ID!
}

type Query {
  companies: [Company]!
  hotels: [Hotel]!
}
```

```ssh
> graphql-inspector validate ./client/**/*.graphql  ./schema.graphql

Error: 
      Unable to find any GraphQL type definitions for the following pointers:
        
          - ./client/hotels.graphql
          
    at prepareResult (/Users/username/.npm/lib/node_modules/@graphql-inspector/cli/node_modules/@graphql-tools/load/index.cjs.js:591:15)
    at loadTypedefs (/Users/username/.npm/lib/node_modules/@graphql-inspector/cli/node_modules/@graphql-tools/load/index.cjs.js:556:12)
    at async Object.loadSchema (/Users/username/.npm/lib/node_modules/@graphql-inspector/cli/node_modules/@graphql-tools/load/index.cjs.js:641:21)
```

2. Second Approach
But, if I create a `documents.graphql`, then it passes.

`documents.graphql`
```
query getCompanies {
  companies {
    id
  }
}
query getHotels {
  hotels {
    id
  }
}
```

```ssh
> graphql-inspector validate ./documents.graphql  ./schema.graphql

success All documents are valid
```

**Expected behavior**

The first case should pass based on the [documentations](https://www.npmjs.com/package/@graphql-inspector/cli)

**Environment:**

- OS: MacOS Big Sur
- `@graphql-inspector/...`: 2.9.0
- `graphql`: 15.5.3
- NodeJS: v14.17.3

**Additional context**

**Comment by gsingh56** [comment]
I am getting the same error. But for me if I run the validate command in the terminal, it works perfectly fine. But when I add it as a npm script in the package.json, it gives the following error: Error: 
      Unable to find any GraphQL type definitions for the following pointers:
        
          - ./schema/graphql/apps/apps.graphqls
          
    at prepareResult (/usr/lib/node_modules/@graphql-inspector/cli/node_modules/@graphql-tools/load/index.cjs.js:591:15)
    at loadTypedefs (/usr/lib/node_modules/@graphql-inspector/cli/node_modules/@graphql-tools/load/index.cjs.js:556:12)
    at async Object.loadSchema (/usr/lib/node_modules/@graphql-inspector/cli/node_modules/@graphql-tools/load/index.cjs.js:641:21)

This is the command I run in the terminal:
graphql-inspector validate 'src/schema/queries/apps/*.graphql' 'src/schema/graphql/apps/*.graphqls'

This is the what I added in the package.json scripts:
"validate": "graphql-inspector validate 'src/schema/queries/apps/*.graphql' 'src/schema/graphql/apps/*.graphqls'"

The upper command works perfectly fine but the lower one fails.

**Comment by kamilkisiela** [comment]
released in v3.0.0

**Comment by FerasAlazzeh** [comment]
I am still seeing this bug on latest version
